config/pod.yaml.tim

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: "{{build_id_with_prefix}}"
+  name: "{{pod_name}}"
```

Comment:
it used to be very easy to describe or see pod logs eg; `kubectl describe pod **12115** --namespace=cd-screwdriver-k8s-build` is that going to change ?

Reply:
We can always do a `k get pods | grep 12115`

Reply:
Could we add another label that's like `build: {{build_id}}`?

Reply:
Nevermind, I see that there's a label for `sdbuild`.